gradle-plugins/compose/src/test/test-projects/application/hotReload/build.gradle

```diff
@@ -0,0 +1,69 @@
+import org.jetbrains.compose.desktop.application.dsl.TargetFormat
+
+plugins {
+    id "com.android.application"
+    id "org.jetbrains.kotlin.multiplatform"
+    id "org.jetbrains.kotlin.plugin.compose"
+    id "org.jetbrains.compose"
+}
+
+kotlin {
+    // empty stub (no actual android app) to detect configuration conflicts
+    // like https://github.com/JetBrains/compose-jb/issues/2345
+    androidTarget()
+
+    jvm()
+    sourceSets {
+        jvmMain {
+            dependsOn(commonMain)
+
+            dependencies {
+                implementation(compose.desktop.currentOs)
+            }
+        }
+    }
+    jvmToolchain {
```

Comment:
It seems it will require JDK 11 to be installed that won't be a case on CI soon.
I'd suggest approach like here - https://github.com/JetBrains/compose-multiplatform/commit/51a87ca55cd5bc7e8b9cc78b6c7c1531bda2d96a